java/src/main/java/com/logicalclocks/hsfs/engine/SparkEngine.java

```diff
@@ -542,4 +553,31 @@ private void setupAdlsConnectorHadoopConf(StorageConnector.AdlsConnector storage
     }
     return emptyDataframe;
   }
+
+  public void streamToHudiTable(StreamFeatureGroup streamFeatureGroup, Map<String, String> writeOptions)
```

Comment:
question: Currently, streaming data is not saved offline using delta streamer yet. Do you plan to save it in this PR?